src/openrct2/world/map.c

```diff
@@ -1796,38 +1798,76 @@ static uint8 map_get_highest_land_height(sint32 xMin, sint32 xMax, sint32 yMin,
     return max_height;
 }
 
+
+static sint32 map_correct_corners(sint32 corner)
+{
+    if (corner < 0)
+    {
+        corner = MAP_SELECT_TYPE_CORNER_3;
+    }
+    return corner;
+}
+
 static money32 raise_land(sint32 flags, sint32 x, sint32 y, sint32 z, sint32 ax, sint32 ay, sint32 bx, sint32 by, sint32 selectionType)
 {
     money32 cost = 0;
+    sint32  corners[2] = { MAP_SELECT_TYPE_EDGE_0, MAP_SELECT_TYPE_EDGE_0 };
 
-    if (selectionType < 0 || selectionType >= countof(map_element_raise_styles))
+    if (selectionType >= MAP_SELECT_TYPE_EDGE_0 && selectionType <= MAP_SELECT_TYPE_EDGE_3) 
+    {
+        sint32 diff = EDGE_TO_CORNERS_ROTATION_OFFSET;
+        //we need to transform edge - check double corner operations; diff cannot be explicit as define due to some optimalisations (different result)
```

Comment:
optimalisations > optimisations
Also needs a space after `//` and capitalisation of the W (I know, I'm being nitpicky here)